src/game/Utils/WordWrap.cc

```diff
@@ -422,19 +491,35 @@ UINT16 IanWrappedStringHeight(UINT16 max_w, UINT8 gap, SGPFont font, const ST::u
 				{ // turn bold OFF
 					cur_font = font;
 				}
+
+				if (*(i + 1) == TEXT_SPACE)
+				{
+					i++;
+				}
 				break;
 
 			case TEXT_CODE_NEWCOLOR:
 			case TEXT_CODE_DEFCOLOR:
+				if (*(i + 1) == TEXT_SPACE)
+				{
+					i++;
+				}
```

Comment:
This repeats a lot, so I suggest you add a static function to replace the calls. `SkipSpace` sounds like an appropriate name.